src/balatro_gym/cards/joker.py

```diff
@@ -120,3 +118,8 @@ def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, score
         if scored_hand == PokerHandType.THREE_SET:
             return 12
         return 0
+
+
+JOKERS: set[JokerBase] = {
```

Comment:
Does this need to hold the objects themselves? Or could this just track the types? Depending on how `JOKERS` gets used, it might lead to weird behavior in the case of repeat jokers?